.husky/pre-commit

```diff
@@ -1,6 +1,3 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
-
```

Comment:
I believe you'll want to do this to https://github.com/adobe/spectrum-web-components/blob/e7b39dc4eb68c01dfcecf458d1493fa16ef6d491/.husky/commit-msg as well.